routes/reviews.js

```diff
@@ -1,15 +1,19 @@
 import express from "express";
 import cmsAdapter from "../src/cmsAdapt.js";
 import { getReviewsSizeFive, postReview } from "../utils/reviewsUtils.js";
+import { getAverageRating } from "../utils/reviewsUtils.js";
 const reviewRouter = express.Router();
 
 reviewRouter.get('/movies/:id/reviews/:page', async (req, res) => {
   const payload = await getReviewsSizeFive(cmsAdapter ,req.params.id, req.params.page)
   res.json(payload);
 });
 
-//Skriv här!
 
+reviewRouter.get('/movies/:id/ratings', async (req, res) => {
+const payload = await getAverageRating( req.params.id);
```

Comment:
I find indentation to be off on this line. I would have expected it to be indented with two space, to match the next line. I also find the extra space after the `(` parenthesis to be inconsistent with how you've written the same type of code elsewhere.